evalml/tests/conftest.py

```diff
@@ -372,43 +372,38 @@ def fit(self, X, y):
 
 
 @pytest.fixture
-def dummy_classifier_linear_component_graph(dummy_classifier_estimator_class):
-    component_graph_linear = {
-        "Name": ["Imputer", "One Hot Encoder", dummy_classifier_estimator_class]
-    }
-    return component_graph_linear
-
-
-@pytest.fixture
-def dummy_regressor_linear_component_graph(dummy_regressor_estimator_class):
-    component_graph_linear = {
-        "Name": ["Imputer", "One Hot Encoder", dummy_regressor_estimator_class]
-    }
-    return component_graph_linear
-
-
-@pytest.fixture
-def dummy_classifier_dict_component_graph(dummy_classifier_estimator_class):
```

Comment:
Removed these in favor of `example_graph` and `example_regression_graph` since they were confusing; they used dummy classifier but were labeled as Random Forest. Didn't provide much, if any value over the example_graph fixture